lib/data/minimum_wage_data.yml

```diff
@@ -136,7 +136,10 @@
     :max_age: 21
     :rate: 5.30
   - :min_age: 21
-    :max_age: 1000
+    :max_age: 24
```

Comment:
The `MarriageAbroadCalculator#per_hour_minimum_wage` method uses `@age < r[:max_age]` which means this rate isn't picked up for people aged 24. I've created PR #2441 to fix this problem.